src/MARIGOLD/Condition.py

```diff
@@ -1895,6 +1895,21 @@ def calc_mu_eff(self, method='Ishii', alpha_max = 1.0):
 
         return
     
+    # Test different ways of calculating mixture viscosity, 05/15 Quan

```

Comment:
Why is this not just an option for calc_me_eff?

Ishii_AA might be a good name for it. Would have to add it to every entry in the dictionary tho